emacs/merlin.el

```diff
@@ -1979,9 +1982,11 @@ Empty string defaults to jumping to all these."
     (define-key merlin-map (kbd "C-c C-x") #'merlin-error-next)
     (define-key merlin-map (kbd "C-c C-l") #'merlin-locate)
     (define-key merlin-map (kbd "C-c &"  ) #'merlin-pop-stack)
-    (define-key merlin-map (kbd "C-c C-r") #'merlin-error-check)
+    (define-key merlin-map (kbd "C-c C-v") #'merlin-error-check)
     (define-key merlin-map (kbd "C-c C-t") #'merlin-type-enclosing)
-    (define-key merlin-map (kbd "C-c C-d") #'merlin-destruct)
+    (define-key merlin-map (kbd "C-c C-d") #'merlin-document)
+    (define-key merlin-map (kbd "C-c M-d") #'merlin-destruct)
+    (define-key merlin-map (kbd "C-c |") #'merlin-destruct)
```

Comment:
How does that work on macos ? The `|` character is a bit hard to type already (`alt+maj+l`)

Reply:
Indeed: this `|`-based binding (especially chosen to suggest the pattern-matching construct) is easier to type on QWERTY and AZERTY keyboards! I guess macOS users would rather use the other keybinding for `merlin-destruct`…

So maybe, this could be another argument to keep these two bindings for `merlin-destruct`?
(see also [this previous comment](https://github.com/ocaml/merlin/pull/1668#issuecomment-1709132411) on that point)

Anyway, feel free to let us know if you're also OK with this dual-binding idea… or if you'd prefer that I remove `C-c |` to only keep `C-c M-d`.
Both choices would be fine with me.

Reply:
The dual-binding seems fine.